raspberry_model.py

```diff
@@ -0,0 +1,69 @@
+from ner_processing.master_mapping import DATA_IDS
+from ner_processing.master_mapping import MESSAGE_IDS
+from ner_processing.message import Message
+from typing import Optional
+import can
```

Comment:
This doesnt compile if your own mac

Reply:
it definitely does you just need to install the dependency